### PR TITLE
Alert

### PR DESCRIPTION
**Pull request by sanjay-xdr**
added alert component
structured the file

### CODE REVIEW

**Comment by sanjay-xdr** [comment]
alert-cl

**Comment by Koushith** [comment]
@sanjay-xdr This looks good, but try reducing the font size.

File: Utility CSS/utility.css [hunk 1]
```diff
@@ -100,3 +100,6 @@
 .margin-m1 {
   margin: 4px 6px;
 }
+.margin-m2 {
+  margin: 8px 10px;
+}
```
Comment (Koushith) [inline Utility CSS/utility.css:105]: Try to use relative units here.

Comment (sanjay-xdr) [reply Utility CSS/utility.css:105]: hey thanks for the review but as vikram said below 10px you should not use rem so i used px

**Review by NtshVrm — APPROVED** [review]

**Pull request merged**